book/website/project-design/sdp/pd-sdp-resources.md

```diff
@@ -0,0 +1,22 @@
+(pd-sdp-resources)=
+
+# Resources
+
+## Keeping sensitive files secure
+
+For more information about`.gitignore`, please check out the following resources:
+
+* Git `.gitignore` [documentation](http://git-scm.com/docs/gitignore)
+* Pro Git book's section on [ignoring files](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring)
+* GitHub Docs's section on [ignoring files](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files)
+* Atlassian's [tutorial on `.gitignore`](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)
+* freeCodeCamp's blog post [*Gitignore Explained: What is Gitignore and How to Add it to Your Repo*](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/)
+* Linuxize's blog post on [*Ignoring Files and Directories in Git (.gitignore)*](https://linuxize.com/post/gitignore-ignoring-files-in-git/)
+
+## Sharing your Jupyter notebook
+
+For more information about using`nbstripout`, please check out the following resources:
```

Comment:
```suggestion
For more information about using `nbstripout`, please check out the following resources:
```
Just a little typo because we saw it together 👾